### DISCUSSION TITLE
Allow selection of which tab to open on

### DISCUSSION THREAD

**Discussion by andrewtavis** [community/community#68727] [Mobile]
A settings option I was expecting to see in the mobile app was a simple one to say which tab I'd like the app to open on. Not sure how helpful others would find this, but - being used to GitHub mobile web - I'm used to always starting from the feed and jumping off from there. Would be nice to have an option where I can say always open on `Explore` :)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.132.0 (110686114)
</details>

**Comment by taki-on** [comment]
Hi @andrewtavis. GitHub iOS remembers which tab users were on when they closed the app. If you closed (i. e. killed) the app when you were in the Explore tab, then the app should start on the Explore tab when you re-open it. If it does not work for you, it might be a bug. Would you mind giving us some more information about when you experienced the issue?

**Reply by andrewtavis** [reply]
Hey @taki-on :) Thanks for checking in 🙏

The behavior does function as you described it. I close the app, and then on re-open I'm at the same tab. My point was it'd be nice to be able to choose where I start by getting an option to set which tab it opens to so that I can have a consistent user experience each time.

**Reply by taki-on** [reply]
Aaah! Thank you for the clarification. It is an interesting idea! I will pass it to our design folks to investigate!

**Reply by andrewtavis** [reply]
Thanks @taki-on! Hope the suggestion is of use :) I understand if it's not a priority though!
